src/sdk/model/o365/__json__/o365-modify-credentials-request-json.ts

```diff
@@ -4,4 +4,9 @@
 export interface O365ModifyCredentialsRequestJson {
   user: string;
   password: string;
+  use_modern_auth: boolean;
+  application_id: string;
+  application_secret: string;
+  is_exchange_online: boolean;
```

Comment:
according to the swagger doc none of the three boolean properties on this request object are required (they don't have a red asterisk). it would be good to test the update request with & without `use_modern_auth`, `is_exchange_online`, and `is_share_point_online` fields to see if they are truly optional